lib/game_classes/Game.js

```diff
@@ -153,6 +172,27 @@ Game.prototype.addImpassable = function (distanceFromTop, distanceFromLeft) {
     }
 };
 
+// Adds a spawn point to map with (S1, S2) as subTypes
+Game.prototype.addSpawnPoint = function (distanceFromTop, distanceFromLeft, spawnValue) {
+    if (this.hasStarted) {
+        throw new Error('Cannot add impassables after the game has started!');
+    }
+    // Can only add an spawn point to unoccupied spaces are not spawn points 
+    if (
+        this.board.tiles[distanceFromTop][distanceFromLeft].type === 'Unoccupied' &&
+        this.board.tiles[distanceFromTop][distanceFromLeft].subType !== 'SpawnPoint'
+    ) {
+    // Creates new Unoccupied object with spawn value
+        var spawnPoint = new Unoccupied(distanceFromTop, distanceFromLeft, spawnValue);
+
+    // Puts impassable on board
+        this.board.tiles[distanceFromTop][distanceFromLeft] = spawnPoint;
```

Comment:
Since you're accessing `this.board.tiles[distanceFromTop][distanceFromLeft]` three times in this method, it makes sense to throw it into a variable higher up. That way, the `if` condition can probably be short enough to stay on one line instead of split into 4.